dockers/asset-publisher/Dockerfile

```
@@ -1,11 +1,11 @@
-FROM golang:1.14.6-alpine3.12 AS builder
+FROM golang:1.17.3-alpine3.15 as builder
 WORKDIR /tool
 COPY main.go .
 COPY go.mod .
 COPY go.sum .
```

[nits] It;s seems better to reduce the number of layers ASAP.
```suggestion
COPY main.go go.mod go.sum .
```

---

oh, it's too late👼

---

Oops. Nice catch! Let me fix them in another PR.